data/json/items/armor/jewelry.json

```
@@ -1111,7 +1111,381 @@
+        "color": "yellow",
+        "weight": 15
+      },
+      {
+        "id": "celticcross_necklace",
+        "name": { "str": "Celtic Cross necklace" },
+        "description": "A necklace made of a fine gold chain with the Celtic Cross symbol as a pendant.  It is a symbol a variety of Christian faith denominations, such as the Anglicans or Presbyterians.  Featuring a nimbus or ring that emerged in Ireland, France and Great Britain in the Early Middle Ages, it is a type of ringed cross.  It became widespread through its use in the stone high crosses erected across the islands, especially in regions evangelized by Irish missionaries, from the ninth through the 12th centuries.",
+        "color": "yellow",
+        "weight": 20
+      },
+      {
+        "id": "presbyterian_necklace",
+        "name": { "str": "Presbyterian Cross necklace" },
+        "description": "A necklace made of a fine gold chain with the Presbyterian Cross symbol as a pendant.  It is a symbol of the Presbyterian Christian faith.  The Presbyterian Cross emphasize the resurrection and that Christ is not continually dying, but died once and is alive for all eternity, alongside the flames, which represent the Holy Spirit.",
+        "color": "yellow",
+        "weight": 10
+      },
+      {
+        "id": "anglican_necklace",
+        "name": { "str": "Compass Rose necklace" },
+        "description": "A necklace made of a fine gold chain with the Golden Compass Rose symbol as a pendant.  It is a symbol of the Anglican Christian faith.  Anglicanism was the State Religion of the United Kingdom before the Cataclysm.  The compass symbolizes the worldwide spread of the Anglican Faith.  Surmounting the shield, at the North, is a mitre, the symbol of apostolic order essential to all Churches and Provinces constituting the Anglican Communion.",
+        "color": "yellow",
+        "weight": 10
+      },
+      {
+        "id": "general_necklace",
+        "name": { "str": "cross necklace" },
+        "description": "A necklace made of a fine gold chain with a plain cross symbol as a pendant.  It is a symbol of Christian faith, no matter the denomination.",
+        "color": "yellow",
+        "weight": 35
+      },
+      {
+        "id": "episcopal_necklace",
+        "name": { "str": "Episcopal Shield necklace" },
+        "description": "A necklace made of a fine gold chain with the Episcopalian Shield symbol as a pendant.  It is a symbol of the Episcopalian Christian faith.  The shield of the Episcopal Church was adopted by its General Convention in 1940 and depicts the red cross of St. George, patron saint of England, dividing the silver field of the English flag into four rectangles.  The colours red, white, and blue are also, coincidentally, the colours of the flags of both the United States and the United Kingdom of Great Britain and Northern Ireland.",
```

```suggestion
        "description": "A necklace made of a fine gold chain with the Episcopalian Shield symbol as a pendant.  It is a symbol of the Episcopalian Christian faith.  The shield of the Episcopal Church was adopted by its General Convention in 1940 and depicts the red cross of St. George, patron saint of England, dividing the silver field of the English flag into four rectangles.  The colors red, white, and blue are also, coincidentally, the colors of the flags of both the United States and the United Kingdom of Great Britain and Northern Ireland.",
```

We use american spelling